<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>funcDlg</class>
 <widget class="QWidget" name="funcDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>110</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="homeBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pageUpBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PageUp</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="endBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>End</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pageDownBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PageDown</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="delBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="prtScBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PrtSc</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="insertBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Insert</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="scLkBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ScLk</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="noLkBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>NoLk</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="inputMethodBtn">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>中</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
